src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -219,6 +220,9 @@ public DashboardWebApplication(
 
         builder.Services.AddScoped<DimensionManager>();
 
+        builder.Services.AddSingleton<IFileProvider>(new PhysicalFileProvider(Directory.GetCurrentDirectory()));
```

Comment:
why is this necessary?

Reply:
We're the first consumer of the `IFileProvider` abstraction. It allows the other service `DashpageJsonFilePersistence` access to the file system.